modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -2064,12 +2064,49 @@ public int compare(CodegenProperty one, CodegenProperty another) {
         return m;
     }
 
+    private void addComposedMappedModels(CodegenDiscriminator discriminator, ComposedSchema c, String discPropName) {
```

Comment:
It would be good to add function comments and essentially copy the text from the PR to help understand what this is doing.

Reply:
Added a document comment describing what this function is doing